packages/mobile/src/components/lineup-tile/LineupTileCoSign.tsx

```diff
@@ -54,7 +54,7 @@ type Props = {
 }
 
 export const LineupTileCoSign = ({ coSign }: Props) => {
-  const trackTileStyles = useThemedStyles(createTrackTileStyles)
```

Comment:
we should prob look to remove useThemeStyles everywhere at some point

Reply:
I'm down, there are a lot of uses though

Reply:
yeah the whole sign-up flow could use a perf facelift 🗡️